index.html

```diff
@@ -0,0 +1,250 @@
+          </ul>
+          <button class="card-check" type="button">See Project</button>
+        </div>
+      </div> 
+
+      <div class="card-3">
+        <img class="card-image" src="images/Snapshoot-Portfolio-3.svg" alt="card">
+        <div class="card-content">
+          <div class="card-header">
+            <h2>Tonic</h2>
+            <ul class="canopy-list">
+              <li class="canopy">CANOPY</li>
+              <li>Back-End-Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+          </p>
+          <ul class="card-buttons">
+            <li><button type="button">html</button></li>
+            <li><button type="button">css</button></li>
+            <li><button type="button">javascript</button></li>
+          </ul>
+          <button class="card-check" type="button">See Project</button>
+        </div>
+      </div> 
+      
+      <div class="card-4">
+        <img class="card-image" src="images/Snapshoot-Portfolio-4.svg" alt="card">
+        <div class="card-content">
+          <div class="card-header">
+            <h2>Multi-Post Stories</h2>
+            <ul class="canopy-list">
+              <li class="canopy">CANOPY</li>
+              <li>Back-End-Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+          </p>
+          <ul class="card-buttons">
+            <li><button type="button">html</button></li>
+            <li><button type="button">css</button></li>
+            <li><button type="button">javascript</button></li>
+          </ul>
+          <button class="card-check" type="button">See Project</button>
+        </div>
+      </div> 
+      <div id="about-myself">
+        <div class="top">
+          <h2>About<br>Myself</h2>
+          <p>Hello I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>
+          <a href="index.html" class="connect">LET'S CONNECT</a>
+          <div id="media2">
+            <ul id="details2">
+              <li><a href=""><img src="images/Github.svg" alt="Github"></a></li>
+              <li><a href=""><img src="images/LinkedIn.svg" alt="LinkedIn"></a></li>
+              <li><a href=""><img src="images/Angellist.svg" alt="Angellist"></a></li>
+              <li><a href=""><img src="images/Twitter.svg" alt="Twitter"></a></li>
+              <li><a href=""><img src="images/Medium.svg" alt="Medium"></a></li>
+            </ul>
+          </div>
+          <button class="card-check" type="button">Get my resume</button>
+        </div>
```

Comment:
- [ ] Kindly Increase the padding size of the button to match the Figma design. :+1: